vips/vip-304/bscmainnet.ts

```diff
@@ -0,0 +1,240 @@
+  "0xcF6BB5389c92Bdda8a3747Ddb454cB7a64626C63", // XVS XVSTokenConverter BaseAsset
+];
+
+const vip304 = (babyDogeMaxStalePeriodInSeconds: number = STALE_PERIOD) => {
+  const meta = {
+    version: "v2",
+    title: "Meme Pool",
+    description: ``,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+
+  return makeProposal(
+    [
+      // Set Oracle Config for BabyDoge
+      {
+        target: BINANCE_ORACLE,
+        signature: "setMaxStalePeriod(string,uint256)",
+        params: ["BABYDOGE", babyDogeMaxStalePeriodInSeconds],
+      },
+      {
+        target: BINANCE_ORACLE,
+        signature: "setSymbolOverride(string,string)",
+        params: ["BabyDoge", "BABYDOGE"],
+      },
+      {
+        target: RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            BABYDOGE,
+            [
+              BINANCE_ORACLE,
+              "0x0000000000000000000000000000000000000000",
+              "0x0000000000000000000000000000000000000000",
+            ],
+            [true, false, false],
+          ],
+        ],
+      },
+
+      // Swap router
+      {
+        target: SWAP_ROUTER,
+        signature: "acceptOwnership()",
+        params: [],
+      },
+
+      // Add Meme Pool
+      {
+        target: COMPTROLLER,
+        signature: "acceptOwnership()",
+        params: [],
+      },
+      {
+        target: COMPTROLLER,
+        signature: "setPriceOracle(address)",
+        params: [RESILIENT_ORACLE],
+      },
+      {
+        target: POOL_REGISTRY,
+        signature: "addPool(string,address,uint256,uint256,uint256)",
+        params: ["Meme", COMPTROLLER, "500000000000000000", "1100000000000000000", "100000000000000000000"],
+      },
+
+      // Add BabyDoge Market
+      {
+        target: TREASURY,
+        signature: "withdrawTreasuryBEP20(address,uint256,address)",
+        params: [BABYDOGE, BABYDOGE_SUPPLY, NORMAL_TIMELOCK],
+      },
+      {
+        target: BABYDOGE,
+        signature: "approve(address,uint256)",
+        params: [POOL_REGISTRY, 0],
+      },
+      {
+        target: BABYDOGE,
+        signature: "approve(address,uint256)",
+        params: [POOL_REGISTRY, BABYDOGE_SUPPLY],
+      },
+      {
+        target: VBABYDOGE,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["100"],
+      },
+      {
+        target: POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            VBABYDOGE,
+            "300000000000000000",
+            "400000000000000000",
+            BABYDOGE_SUPPLY,
+            TREASURY,
```

Comment:
The receiver of the minted tokens must be `0x866c1B85a0257e8C49a8Bfd3c1Bcc143DDB3EA37`